example-third-party/authentication.py

```diff
@@ -0,0 +1,9 @@
+import urllib.parse
+import secrets
+
+
+def get_url(server: str, lao_id: str, client_id: str) -> str :
+    baseUrl = "https://{}/authorize?response_mode=query&response_type=id_token%20token&client_id={}" \
```

Comment:
Suggestion: you should be able to build the query string from a dictionary, which could make this a lot more readable 😉

Reply:
Done